code/PostgreSQLSchemaManager.php

```diff
@@ -495,7 +495,7 @@ private function alterTableAlterColumn($tableName, $colName, $colSpec)
         // TODO: this returns an empty array for the following string: int(11) not null auto_increment
         //         on second thoughts, why is an auto_increment field being passed through?
 
-        $pattern = '/^([\w(\,)]+)\s?((?:not\s)?null)?\s?(default\s[\w\.\']+)?\s?(check\s[\w()\'",\s]+)?$/i';
+        $pattern = '/^([\w(\,)]+)\s?((?:not\s)?null)?\s?(default\s[\w\.\'\\\\]+)?\s?(check\s[\w()\'",\s\\\\]+)?$/i';
```

Comment:
This regex is meant to split up $colSpec and identify the various parts ofthe column definition. e.g.:
```
varchar(255) default 'Internal' check ("RedirectionType" in ('Internal', 'External', null))
```

This works OK until your enum starts having backslashes, as our ClassName now have.
```
varchar(255) default 'SS\Test\Polymorphic\Player' check ("ClassName" in ('SS\Test\Polymorphic\Player', 'Player', null))
```

All I'm changing here, is allowing the defaults and the list of accepted values to contain backslashes.